src/app/passport/fields.js

```diff
@@ -1,34 +1,51 @@
+const {validators} = require("hmpo-form-wizard/lib/validation");
+
+function firstNameMiddleNameLengthValidator(value, length, firstNameField, middleNameField) {
+  const firstName = this.values[firstNameField];
+  const middleName = this.values[middleNameField];
+
+  const middleNameLength = validators.string(middleName) ? middleName.length : 0;
+  const firstNameLength = validators.string(firstName) ? firstName.length : 0;
+
+  return (middleNameLength + firstNameLength) <= length;
+}
+
+const firstNameMiddleNameLengthValidatorObj = {type: 'firstNameMiddleNameLength' ,  fn:firstNameMiddleNameLengthValidator, arguments: [30, 'firstName', 'middleNames']};
+
 module.exports = {
+  firstNameMiddleNameLengthValidator: firstNameMiddleNameLengthValidator,
```

Comment:
Whats the point of this if we are calling `firstNameMiddleNameLengthValidatorObj` as the validator?

Reply:
I had to include this for the unit test to work